addons/mail/static/src/new/core_ui/message.xml

```diff
@@ -82,7 +82,7 @@
                             <t t-if="message.trackingValues.length">
                                 <t t-call="mail.tracking_values"/>
                             </t>
-                            <div t-if="message.body" class="o-mail-message-body text-break mb-0 bg-view">
+                            <div t-if="message.body" class="o-mail-message-body text-break mb-0 bg-view w-100">
```

Comment:
strange to have to add width instruction on a child that has a flex parent, what does it fix?

Reply:
Without width, It is like this: 
![1](https://user-images.githubusercontent.com/114914773/220290339-e5ec92f6-dddb-46a1-8baf-b075e846ae3b.png)

Reply:
Should be:
![2](https://user-images.githubusercontent.com/114914773/220290623-31973760-5563-4684-9bd9-d39f88742f93.png)